src/app/moves.js

```diff
@@ -0,0 +1,22 @@
+"use server";
+
+/**
+ * @type {Record<string, any[]>}
+ */
+const games = {
+    '123': [
+        { x: 1, y: 2 },
+    ]
+};
+
+export async function addMove(gameId, move) {
+    if (!games[gameId]) {
+        games[gameId] = [];
+    }
+    
+    games[gameId].push(move);
+}
+
+export async function getGames() {
+  return games;
+}
```

Comment:
Server actions are required to be async, which makes sense because they might be called by a client component where they'd need to send data over the network